src/react-native-app/gateways/Api.gateway.ts

```diff
@@ -0,0 +1,150 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+/**
+ * Copied with modification from src/frontend/gateways/Api.gateway.ts
+ *
+ * TODO The React Native example only implements a subset of the functionality defined here, missing in particular is:
+ *  - showing ads
+ *  - showing recommendations
+ *  - calculating shipping costs
+ *  - currency conversion
+ */
```

Comment:
Let's move this one to an issue.

Reply:
@julianocosta89 I had had a question around this above: https://github.com/open-telemetry/opentelemetry-demo/pull/1781#discussion_r1887669025, should it be a standalone issue or a meta issue tracking all React Native follow-ups?

Reply:
I created a meta-tracking [issue](https://github.com/open-telemetry/opentelemetry-demo/issues/1851) for this